ci_group/revolve2/ci_group/morphological_novelty_metric/calculate_novelty.pyi

```diff
@@ -5,4 +5,4 @@ from numpy.typing import NDArray
 
 def calculate_novelty(
     histograms: NDArray[np.int64], amount_instances: int, histogram_size: int
-) -> NDArray[np.float64]: ...
+) -> list[float]: ...
```

Comment:
actually I thought a numpy array was quite nice. what do you think? the only downside is that it currently doesn't have proper typing and stuff but that will change in the future for sure

Reply:
yes I agree, i changed them back now.